app/assets/main/react_components/User/CarouselCategoryButton.jsx

```diff
@@ -51,13 +51,22 @@ const CarouselCategoryButton = ({
   return (
     <>
       {active ? (
-        <button className="button inline-block focus:outline-none m-1 bg-primary text-white">
+        <button
+          className={`${
+            "category_" +
+            categoryName.toLowerCase().replace(/ /g, "_") +
+            "_active"
+          } rounded-full py-1 px-4 button inline-block focus:outline-none text-white m-1`}
+        >
           {" "}
           {categoryName}{" "}
         </button>
       ) : (
         <button
-          className="button inline-block m-1 focus:outline-none"
+          className={` ${
+            "category_" + categoryName.toLowerCase().replace(/ /g, "_")
+          }
+          rounded-full py-1 px-4 border-primary button inline-block m-1 hover:text-white focus:outline-none hover:bg-opacity-80`}
```

Comment:
Perhaps set a default color if there happen to be a category without color to ensure the usability?

Reply:
Yes good idea!